### ISSUE TITLE
Screen proportions remain the same independent of resize

### ISSUE THREAD

**Issue by vickis** [upstage-org/limelight#285]
when you resize the screen there is no change to the proportions of the images (backdrops) on it eg the stage remain proportional in landscape - so if the screen is narrowed then there is white space above and below the image 
everything on the stage remains the same in relation to everything else

**Comment by helenvarley** [comment]
this is great, in that the image is retaining its proportions. however, what's not so great, is that i'm getting white space between the chat window & the image. what happens in UpStage is that the chat window moves with the backdrop. also, the avatars do not scale down - they stay at the same size, which means that they are not going to be in the same place for different audience members. it's fundamental to the whole concept, that everything is in the same proportions for everyone watching. please have a look at one of the stages on live.upstage.org.nz to see what i mean, as it's quite hard to describe it in words or even with screen grabs.

**Comment by thongteav** [comment]
do you mean that the chat window's size and position should stay relevant to the backdrop like that? @helenvarley  
![grouped](https://user-images.githubusercontent.com/26214298/35656367-f0c9b200-075b-11e8-852c-900abfcb4b55.PNG)

**Comment by helenvarley** [comment]
yes. everything should stay relative to everything else. so if you make your window smaller, everything gets smaller. please go onto a stage on live.upstage.org.nz & try making the browser window bigger & smaller, narrower & wider, & you will see how it should work. i guess you need a log in to be able to put some media on the stage & see it, so i will send you this via email.

**Comment by rohrlach** [comment]
@helenvarley is it safe to assume this needs rework?

**Comment by helenvarley** [comment]
yes i think so.

**Comment by thongteav** [comment]
Will be working on this next.

**Comment by rohrlach** [comment]
@thongteav can you remember where we were at with this?

**Comment by thongteav** [comment]
I was stuck on resizing the windows when height changes for the window and the elements won't scale because they are bootstrap.

**Comment by rohrlach** [comment]
Okay thanks.

Don't worry about this issue - I'm going to assign it to the incoming project team, I just couldn't remember where it was at.
